### DISCUSSION TITLE
sidecar.skywalking.apache.org/initcontainer.resources.limits After the annotation is set, the resources resource information is not displayed in the pod

### DISCUSSION THREAD

**Discussion by berylshow** [apache/skywalking#9081] [Q&A] [answered]
After I use the annotation setting
sidecar.skywalking.apache.org/initcontainer.resources.limits: {"memory": "512Mi","cpu": "200m"} sidecar.skywalking.apache.org/initcontainer.resources.requests: {"memory": "512Mi","cpu": "200m"}

 the resources resource information is not displayed in the pod. I'm not sure whether it takes effect
![image](https://user-images.githubusercontent.com/93063206/168799633-c3a14738-ff33-4c2b-b0b5-bffe021971cd.png)

**Comment by dashanji** [comment, marked as answer]
@berylshow Hi, There is a [demo](https://github.com/apache/skywalking-swck/blob/5e7d2879d7a69eed6d3ab30d6b4de957a9449eab/test/e2e/demo.yaml#L31-L33]) about how to use the resource information. Please check if you add the quotes in your annotation as it's a string.
